sdk/storage/azure-storage-file-datalake/azure/storage/filedatalake/_models.py

```diff
@@ -598,3 +598,23 @@ class LocationMode(object):
 
     PRIMARY = 'primary'  #: Requests should be sent to the primary location.
     SECONDARY = 'secondary'  #: Requests should be sent to the secondary location, if possible.
+
+
+class AccessControlChanges(object):
+    """
+    AccessControlChangeResult contains result of operations that change Access Control Lists recursively.
+
+    :param long directories_successful:
```

Comment:
This should be type int - long is no longer a type in Python

Reply:
Also, could we document the public instance variables rather than the parameters?
(i.e. instead of using `:param` we should use `:ivar`